### PR TITLE
Add invalid config warning when cache open write fail and read retry are inconsistent

### PR DESCRIPTION
**Pull request by sudheerv**
are inconsistent

### CODE REVIEW

**Comment by sudheerv** [comment]
@zwoop @ezelkow1 Can you review pls?

File: proxy/http/HttpConfig.h [hunk 1]
```diff
@@ -330,6 +330,16 @@ enum {
   http_stat_count
 };
 
+enum CacheOpenWriteFailAction_t {
+  CACHE_WL_FAIL_ACTION_DEFAULT                           = 0x00,
```
Comment (ywkaras) [inline proxy/http/HttpConfig.h]: Why not just 0, 1 etc. instead of 0x00, 0x01?

Comment (sudheerv) [reply proxy/http/HttpConfig.h]: Hmm..good question. I don’t quite remember why I picked hex numbers. This PR is just moving them around, fwiw. I don’t really have a preference, but  I do have an action item to remove the collapsed forwarding plugin and clean up related configs for 10.0.0 and as part of that I can change them to simple decimals,  unless you feel strongly that I should do it now.

Comment (ywkaras) [reply proxy/http/HttpConfig.h]: No it's not a significant issue.

Comment (sudheerv) [reply proxy/http/HttpConfig.h]: Cool, closing this now, will clean it up in 10.0

**Comment by zwoop** [comment]
Cherry-picked to v9.0.x branch.

**Pull request merged**